doc/rose-rug-advanced-tutorials-bunch.html

```diff
@@ -0,0 +1,333 @@
+      like this:</p>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example suite.rc</h3>
+      <pre class="prettyprint lang-cylc">
+[cylc]
+    UTC mode = True # Ignore DST
+    [[event hooks]]
+        timeout handler = rose suite-hook --mail --shutdown
+        timeout = P3D
+[scheduling]
+    [[dependencies]]
+        graph = lander
+[runtime]
+    [[root]]
+        command scripting = rose task-run
+    [[lander]]
+</pre>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App</h3>
+
+      <p>In the suite directory create an <samp>app/</samp> directory</p>
+      <pre class="shell">
+mkdir app
+</pre>
+
+      <p>In the <var>app</var> directory create a <var>lander/</var>
+      directory</p>
+      <pre class="shell">
+cd app
+mkdir lander
+</pre>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App (2)</h3>
+
+      <p>In the <var>app/lander/</var> directory create a
+      <samp>rose-app.conf</samp> file. Using your editor of choice, paste the
+      following lines into it:</p>
+      <pre class="prettyprint lang-cylc">
+mode=rose_bunch
+
+[bunch]
+command-format=land %(class)s %(passengers)s
+
+[bunch-args]
+class=airbus concorde airbus cessna
+passengers=40 20 30 2
+</pre>
+
+      <p>This configuration will run a <code>rose bunch</code> task that calls
+      multiple instances of the <code>land</code> command, supplying arguments
+      to each instance from the <var>class</var> and <var>passengers</var>
+      entries under <var>[bunch-args]</var>.</p>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App (3)</h3>
+
+      <p>In the <var>app/lander/</var> directory create a <samp>bin/</samp>
+      directory. Using your editor of choice, create a file named "land" under
+      the bin directory and paste in <a href=
+      "etc/rose-rug-advanced-tutorials-rose-bunch/src/land.html">these</a>
+      lines. This script capures the landing routine and expects two arguments;
+      the plane type (its class) and the number of passengers it is
+      carrying.</p>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App (4)</h3>
+
+      <p>Finally, make the new <samp>land</samp> file executable by navigating
+      into the bin directory of the <samp>lander</samp> app and running:</p>
+      <pre class="shell">
+chmod +x land
+</pre>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Run Example</h3>
+
+      <p>Navigate to the root directory of your suite and run <samp>rose
+      suite-run</samp>. Your suite should run, launch the cylc gui and
+      successfully run the lander app.</p>
+
+      <p>Once the suite has finished running and has shutdown, open <samp>rose
+      bush</samp> to view its output. You can quickly get to the relevent page
+      by running <samp>rose slv</samp> in the suite directory.</p>
```

Comment:
`suite directory` here, `root directory` above, I would prefer that it was consistent, or if you must `root directory, also referred to as the suite directory where the suite.rc and the rose-suite.conf are located`